src/sidebar/components/user-menu.js

```diff
@@ -86,6 +101,12 @@ function UserMenu({ auth, bridge, onLogout, serviceUrl, settings }) {
               href={serviceUrl('account.settings')}
             />
           )}
+          {isNotebookEnabled && (
+            <MenuItem
+              label="Open notebook"
+              onClick={() => onSelectNotebook()}
```

Comment:
While testing this I noticed an existing issue where menu items that are just links show a pointer when their label is hovered whereas menu items that are not links show an I-Beam when the cursor is hovered over the text. That's an inconsistency we should fix at some point.